remove references to Marketplace and High Fidelity (#162)
vircadia/vircadia-docs-sphinx: docs/source/create/avatars/package-avatar.rst

Package and Host Your Avatar
##################################

> At a minimum, avatars in Vircadia must have an FBX, glTF, or GLB model, and an associated FST file that includes information about how your avatar looks and behaves. Together, these two files (with any optional texture or script) form an "avatar package". There are two ways you can create an avatar package: by using the `Avatar Packager`_ in Interface or the `High Fidelity Avatar Exporter for Unity`_ in Unity.

Reviewer: ```suggestion
At a minimum, avatars in Vircadia must have an FBX, glTF, or GLB model, and an associated FST file that includes information about how your avatar looks and behaves. Together, these two files (with any optional texture or script) form an "avatar package". There are two ways you can create an avatar package: by using the `Avatar Packager`_ in Interface or the `Vircadia Avatar Exporter for Unity`_ in Unity.
```

Reviewer: Just a warning, this package is not being actively maintained by us (but anyone is free to take it on if they want to.)

Reviewer 2: You link to the "Vircadia Avatar Exporter" but further up you still talk about "The High Fidelity Avatar Exporter for Unity"

Reviewer: Fixed.

Package Your Avatar
---------------------------

If you're reading this page, you likely already :doc:`built your own FBX model <create-avatars>` or :doc:`found and downloaded a model <find-avatars>` that you want to use in Vircadia. Therefore, all that remains is to package your avatar and create the FST file. This file includes information about the skeleton, blendshapes, textures, and scripts used by your avatar.

We provide two ways to create an avatar package: either through Unity or through our Avatar Packager.

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> High Fidelity Avatar Exporter for Unity

Reviewer: ```suggestion
Vircadia Avatar Exporter for Unity (unmaintained)
```

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> In some cases, you will want to :doc:`download an avatar from an external website <find-avatars>` and use that avatar in Vircadia. The High Fidelity Avatar Exporter for Unity (also known as the "avatar exporter") converts human-like avatars and packages them for use in Vircadia. 

Reviewer: ```suggestion
In some cases, you will want to :doc:`download an avatar from an external website <find-avatars>` and use that avatar in Vircadia. The Vircadia Avatar Exporter for Unity (also known as the "avatar exporter") converts human-like avatars and packages them for use in Vircadia. 
```

> Once you have successfully used the :doc:`avatar exporter <find-avatars>` to package your avatar, you must host it somewhere on the cloud. You can host it on our servers (using the Avatar Packager), or simply copy the avatar package to your own servers. 

Reviewer: ```suggestion
Once you have successfully used the :doc:`avatar exporter <find-avatars>` to package your avatar, you must host it somewhere on the cloud. You can upload it to Amazon S3 or a webserver for example.
```

   * Texture Folder: If your avatar has textures in a separate folder, specify the folder location. If your avatar's textures are embedded in the FBX, you do not need to specify anything. 
4. Click 'Create'.

At this point, you have successfully packaged your avatar. You can close the Avatar Packager here and upload your FST file and FBX model to the cloud location of your choice. 

---------------------------
Host Your Avatar
---------------------------

Before you can use a custom avatar, you must first host its FST and FBX files in a place that is publicly accessible. You can use any cloud platform including Amazon S3, Google Cloud Storage, Microsoft Azure, Dropbox, etc.

> If you want to upload it to High Fidelity's servers or sell your avatar on the Marketplace, use the Avatar Packager:

Reviewer 2: Missed this section.

Troubleshooting with the Avatar Packager 
-------------------------------------------

The Avatar Packager will notify you of any errors or warnings that may affect the way your avatar looks and behaves in Vircadia. This is a list of the errors you may encounter, along with basic instructions on how to fix your avatar. **Errors** (in red) must be fixed before you upload your avatar, while **Warnings** (in orange) may or may not affect whether your avatar will show up and behave correctly in Vircadia.

.. note:: 

    Many of the errors you will encounter describe issues with the avatar's skeleton. The troubleshooting tips below will attempt to fix the errors in Unity. 
    

>     However, if the bone structure of the model does not resemble a humanoid skeleton (with two legs, two arms, hips, chest, spine, and head), then it is likely not compatible with Vircadia. You will not be able to fix these avatars in Unity alone. Instead, you will likely need advanced knowledge of building, rigging, and mapping bones in a 3D modeling tool such as Blender or Maya. 

Reviewer 2: ```suggestion
    However, if the bone structure of the model does not resemble a humanoid skeleton (with two legs, two arms, hips, chest, spine, and head), then it is likely not compatible with Vircadia out of the box. You will not be able to fix these avatars in Unity alone. Instead, you will likely need advanced knowledge of building, rigging, and mapping bones in a 3D modeling tool such as Blender or Maya. 
```

You can use non-humanoids as long as you provide your own animations as far as I know.

            <tr>
                <td>
                    <p id="unsupported-format" style="color: orange;"><strong>Unsupported avatar model format</strong></p>

>                     <p>This warning occurs when your avatar model is not a supported format. Vircadia only supports FBX models for avatars.

Reviewer 2: ```suggestion
                    <p>This warning occurs when your avatar model is not a supported format. Vircadia only supports FBX, glTF, and GLB models for avatars.
```
Pretty sure that the avatar packager has not been updated for this, but Vircadia supports glTF and GLB for avatars as well as FBX nowadays.

                </td>
                <td>

>                     <p>This warning cannot be resolved in Unity or Vircadia. To fix it, you need to open your model in the 3D modeling tool of your choice, and export your model as an FBX model. </p>

Reviewer 2: ```suggestion
                    <p>This warning cannot be resolved in Unity or Vircadia. To fix it, you need to open your model in the 3D modeling tool of your choice, and export your model as an FBX, glTF, or GLB file. </p>
```
